### PR TITLE
update other dependencies

### LINKED ISSUE #220: update other dependencies
These dependencies were not updated within #218:

- symfony
- cs-fixer
- codesniffer
- monolog

### PR DESCRIPTION
**Pull request by mimmi20**
Fixes #220.

### CODE REVIEW

File: src/Browscap.php [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <?php
-declare(strict_types = 1);
+/**
```
Comment (asgrim) [inline src/Browscap.php]: We really don't need a comment for every file :/

File: src/Browscap.php [hunk 2]
```diff
@@ -44,8 +52,8 @@
     /**
      * Browscap constructor.
      *
-     * @param \Psr\SimpleCache\CacheInterface  $cache
-     * @param LoggerInterface $logger
+     * @param \Psr\SimpleCache\CacheInterface $cache
+     * @param LoggerInterface                 $logger
```
Comment (asgrim) [inline src/Browscap.php]: Please let's avoid adding space unnecessarily (no need to align arguments like this, just creates noisy diffs)

File: src/Browscap.php (hunk 1 above)
Comment (mimmi20) [reply src/Browscap.php]: fixed

File: src/Browscap.php (hunk 2 above)
Comment (mimmi20) [reply src/Browscap.php]: fixed

**Pull request merged**